.about.yml

```diff
@@ -111,6 +115,10 @@ services:
   category: security
   url: https://hakiri.io/github/18F/micropurchase/master
   badge: https://hakiri.io/github/18F/micropurchase/master.svg
+- name: Travis
+  category: Continuous Integration
+  url: https://travis-ci.org/18F/micropurchase
+  badge: https://api.travis-ci.org/18F/micropurchase.svg
```

Comment:
maybe for this we can do master branch since that is what is on production? https://travis-ci.org/18F/micropurchase.svg?branch=master (I think this one you have will return badge for whatever the last travis run was...which means it will be red sometimes)